core/amber/src/main/resources/python_udf/texera_udf_server_main.py

```diff
@@ -136,9 +137,21 @@ def do_action(self, context, action):
             # close UDF
             self.udf_op.close()
             yield self._response(b'Success!')
+
+        elif action.type == "terminate":
```

Comment:
again please add some comments to explain how you are releasing threads. Mention the APIs that the Amber actors calls when terminating